### PR TITLE
minor updates to sync repo with what was submitted to arXiv

### PR DESCRIPTION
**Pull request by msneubauer**

### CODE REVIEW

File: CWP/papers/HSF-CWP-2017-05_analysis/latex/cwp-chapters.bib [hunk 1]
```diff
@@ -0,0 +1,243 @@
+% Copyright (C) 2017, The HSF Community White Paper authors, licence CC-BY-4.0.
```
Comment (graeme-a-stewart) [inline CWP/papers/HSF-CWP-2017-05_analysis/latex/cwp-chapters.bib]: There should only be one copy of this file, in roadmap directory, i.e., `HSF-CWP-2017-01_roadmap`, and shared.

Can you take this out please.

File: CWP/papers/HSF-CWP-2017-05_analysis/latex/cwp-wg-analysis-interpretation.tex [hunk 2]
```diff
@@ -11,8 +11,8 @@
 \usepackage[utf8]{inputenc}
 
 \usepackage[style=numeric-comp,sorting=none]{biblatex}
-\addbibresource{../../HSF-CWP-2017-01_roadmap/latex/cwp.bib}
-\addbibresource{../../HSF-CWP-2017-01_roadmap/latex/cwp-chapters.bib}
+\addbibresource{cwp.bib}
```
Comment (graeme-a-stewart) [inline CWP/papers/HSF-CWP-2017-05_analysis/latex/cwp-wg-analysis-interpretation.tex]: Why did this change from the shared `.bib` file? We should only have one shared resource for reference.

File: CWP/papers/HSF-CWP-2017-05_analysis/latex/cwp.bib [hunk 3]
```diff
@@ -0,0 +1,2205 @@
+% Copyright (C) 2017, The HSF Community White Paper authors, licence CC-BY-4.0.
```
Comment (graeme-a-stewart) [inline CWP/papers/HSF-CWP-2017-05_analysis/latex/cwp.bib]: Should not be here - use common copy.

**Comment by msneubauer** [comment]
This PR now includes removal of the local bib files and corresponding change to the main tex file

**Comment by graeme-a-stewart** [comment]
Thanks @msneubauer - merged now.

**Pull request merged**